src/build/tasks/theme-json-schema.ts

```diff
@@ -66,6 +66,7 @@ const tokensSchema: GenericSchema = {
     '^color-': getTokenSchema(getComplexValueSchema(colorValueSchema, visualModes)),
     '^font-family-': getTokenSchema(stringValueSchema),
     '^border-radius-': getTokenSchema(spaceValueSchema),
+    '^border-width-': getTokenSchema(spaceValueSchema),
```

Comment:
You probably want to create a new schema to use here since border-width doesn't accept %